### PR TITLE
XW-1877 | Search suggestions in Global Navigation

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/XW-1877
https://github.com/Wikia/design-system-reference-page/pull/69

### CODE REVIEW

File: components/_global-navigation.scss [hunk 1]
```diff
@@ -199,6 +204,15 @@ $notification-counter-size: 18px;
 		transform: rotate(180deg);
 	}
 
+	&__search-suggestions {
+		top: calc(100% + 1px);
+		width: 100%;
```
Comment (kvas-damian) [inline components/_global-navigation.scss]: Nice read about using `width: 100%` vs `left/top: 0`: http://stackoverflow.com/a/16600540

**Comment by kvas-damian** [comment]
LGTM

File: components/_global-navigation.scss (hunk 1 above)
Comment (rogatty) [inline components/_global-navigation.scss]: Missed it, fixed, thanks.

**Comment by Grunny** [comment]
LGTM.

**Comment by Warkot** [comment]
LGTM

**Comment by hakubo** [comment]
LGTM

**Pull request merged**